File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, s: str) -> bool:
        
        if len(s) < 26:
            return False
        
        no_dupes = "".join(set(str(s)))
        occ = {}
        
        for char in no_dupes:
            occ[char] = 1 + occ.get(char, 0)

        return len(occ.keys()) == 26
    

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    let occ = new Object();
    
    for(i = 0; i < sentence.length; i++){
        occ[sentence[i]]++;
    }
    
    return (Object.keys(occ).length === 26)
};